gettsim/taxes/zu_verst_eink/vorsorgeaufw.py

```diff
@@ -254,7 +254,7 @@ def _vorsorgeaufw_vom_lohn_bis_2019_single(
     bruttolohn_m: float,
     eink_st_abzüge_params: dict,
 ) -> float:
-    """Calcaulate vorsoge expenditures until 2019 for singles.
+    """Calculate vorsoge expenditures until 2019 for singles.
```

Comment:
```suggestion
    """Calculate precautionary expenditures until 2019 for singles.
```
or "Vorsorge", but capitalised and without typo :smile:, same below.